timescaledb/getting-started/add-data.md

```diff
@@ -1,53 +1,63 @@
+stock trade data (aka. "tick data") provided by [twelvedata][twelve-data] for 100 stock symbols for the most recent month. 
 
-## About the dataset
+### About the dataset
 
-Our dataset comes from [OpenWeatherMap](https://openweathermap.org) and contains
-measurements for 10 cities in which Timescalers reside: New York City, San
-Francisco, Princeton, Austin, Stockholm, Lisbon, Pietermaritzburg, Nairobi,
-Toronto, and Vienna.
+The dataset contains tick level (second by second) stock trade data for the top 100 most traded symbols along with a separate table of company symbols and company names. 
+There are two tables that you will ingest, a hypertable `stocks_real_time` and a regular table `company`. 
 
-The dataset contains weather metrics for each city from 1 January 1979 to 27 April 2021.
+Details on the data set are as follows:
+`stocks_real_time`: Stock data indicating quotes for stock prices for a given second during trading hours
 
-For each city, we record the following metrics:
 ```bash
-* time: Timestamp of data calculation
-* timezone: Shift in seconds from UTC
-* city_name: City name
-* temp_c: Temperature in degrees celsius
-* feels_like_c:  This temperature parameter accounts for the human perception of weather
-* temp_min_c: Minimum temperature
-* temp_max_c Maximum temperature
-* pressure_hpa: Atmospheric pressure (at sea level) in hPa
-* humidity_percent:  Humidity as a percentage of maximum water vapor
-* wind_speed_ms: Wind speed in meters per second
-* wind_deg: Wind direction, degrees (meteorological)
-* rain_1h: Rain volume for the last hour, mm
-* rain_3h: Rain volume for the last 3 hours, mm
-* snow_1h: Snow volume for the last hour, in mm
-* snow_3h: Snow volume for the last 3 hours, in mm
-* clouds_percent: Cloudiness as a percentage
-* weather_id: Weather condition id
+    * time: timestamp column incrementing at a tick level (second by second)
+    * symbol: symbols representing a company, mappings found in the `company` table
+    * price: stock quote price for a company at the given timestamp
+    * day_volume: how many shares are traded each day, NULL values indicate the market is closed
 ```
 
-## Accessing the dataset
+`company`: mapping for symbols to company names
 
-We provide a CSV file with commands for inserting the data into your TimescaleDB instance.
+```bash
+    * symbol: the symbol representing a company name
+    * name: corresponding company name
+```
 
-Download the CSV file (in ZIP format) below and insert it into your database from psql.
+<procedure>
 
-Download CSV: <tag type="download">[weather_data.zip](https://s3.amazonaws.com/assets.timescale.com/docs/downloads/weather_data.zip)</tag>
+### Accessing the dataset
 
-After unzipping the file, use the following command (which assumes `weather_data.csv` is located in your current directory):
+1.  Before ingesting the data, you must create another table, `company`. Run the 
+    following SQL to do this. 
```

Comment:
```suggestion
1.  Before ingesting data, create another table named `company`. At the `psql` prompt, run:
```

Going to be more explicit about what is run where because they need to switch between `psql` and command prompt

Reply:
that's an excellent call